geschaeftsordnung-vollversammlung.md

```diff
@@ -0,0 +1,82 @@
+
+## § 1 Geltungsbereich
+
+Die Geschäftsordnung gilt für die Vollversammlung der Fachschaft Digital Engineering an der Universität Potsdam.
+
+## § 2 Tagesordnung
+
+(1) Die Vollversammlung folgt einer Tagesordnung.
+
+(2) Die Tagesordnung muss Teil der fristgerechten Einladung einer Vollversammlung sein.
+
+(3) Bis 14 Tage vor einer Vollversammlung können die einladenden Personen gestellte Anträge in die Tagesordnung aufnehmen.
+
+(4) Bis 7 Tage vor einer Vollversammlung muss die finale Tagesordnung auf dem gleichen Weg wie die Einladung zur Vollversammlung bekanntgegeben werden.
+
+## § 3 Versammlungsleitung
+
+(1) Die Vollversammlung wird von den einladenden Personen geleitet.
+
+(2) Die Versammlungsleitung kann für einzelne Tagesordnungspunkte an andere Mitglieder der Fachschaft übergeben werden.
+
+(3) Die Versammlungsleitung kann bei Bedarf eine Redeliste führen und/oder die Redezeit begrenzen.
+
+(4) Die Versammlungsleitung entscheidet, auf welche Weise Stimmen gezählt oder geschätzt und ob sie anschließend protokolliert werden.
+
+## § 4 Protokolle
+
+(1) Die Vollversammlung dokumentiert ihre Beschlüsse in einem Protokoll.
+
+(2) Die Versammlungsleitung beauftragt ein Mitglied der Fachschaft mit der Protokollführung.
+
+(3) Das Protokoll wird der Fachschaft spätestens 7 Tage nach der Vollversammlung zugänglich gemacht.
+
+## § 5 Anträge
+
+(1) Anträge sind schriftlich zu verfassen und innerhalb von 14 Tagen nach Zugang der Einladung zur Vollversammlung bei den einladenden Personen einzureichen.
+
+(2) Die einladenden Personen entscheiden über die Aufnahme des Antrags in die Tagesordnung.
+
+(3) Anträge können vor der Behandlung durch die antragstellende Person zurückgezogen werden.
+
+## § 6 Änderungsanträge
+
+(1) Änderungsanträge werden vor der Abstimmung des jeweiligen Antrags behandelt.
+
+(2) Änderungsanträge können von der den ursprünglichen Antrag stellenden Person direkt angenommen werden. Geschieht dies nicht, stimmt die Vollversammlung über die Annahme ab.
+
+(3) Änderungsanträge können vor der Vollversammlung schriftlich bekanntgegeben und begründet werden.
+
+## § 7 Anträge zur Geschäftsordnung
+
+(1) Anträge zur Geschäftsordnung werden durch das Heben beider Arme angezeigt. Davon kann abgewichen werden, wenn die Intention für die Versammlungsleitung eindeutig erkennbar ist.
+
+(2) Anträge zur Geschäftsordnung werden unverzüglich behandelt.
+
+(3) Vor der Abstimmung darf je eine antragsberechtigte Person für bzw. gegen den Antrag das Wort erhalten.
+
+(4) Geschäftsordnungsanträge bedürfen einer Zweidrittelmehrheit der anwesenden Mitglieder der Vollversammlung.
+
+(5) Geschäftsordnungsanträge sind:
+  1. Antrag auf Vertagung eines Tagesordnungspunktes (einfache Mehrheit),
+  2. Antrag auf den Schluss der Redeliste,
+  3. Antrag auf den Schluss der Beratung und ggf. sofortige Abstimmung,
+  4. Antrag auf Ende der Sitzung,
+  5. Antrag auf Zählung und/oder Protokollierung des Abstimmungsergebnisses (auf Verlangen einer Person).
```

Comment:
~Bedeutet "(auf Verlangen einer Person)" hier, dass von Absatz 4 abzuweichen ist?~

NVM, gerade den einsamen Absatz unten gelesen ;)

Da das aber immer noch mMn leicht verwirrend ist, wäre es vielleicht besser, den Antrag auf ordnungsgemäße Auszählung nicht als GO-Antrag darzustellen (wegen des möglichen Missverständnisses), sondern als Absatz 4 zu § 8, denn es passt ja zu den Grundsätzen von Abstimmungen und Wahlen.

Gleichzeitig stellt sich IMO die Frage, ob die Veränderung der TO nach ihrer Annahme mit einfacher Mehrheit erfolgen können soll. Was passiert bspw. wenn die Versammlung mit einfacher Mehrheit die Absetzung eines TOP beschließt? Damit hat man den gleichen Effekt wie: Nr. 2, 3 und Ablehnung des Antrags. Mit einfacher Stimmmehrheit. Daher würde ich sagen, die angenommene TO sollte auch durch 2/3-Mehrheit geschützt werden.

Reply:
Nicht alle Abstimmungen werden durch die Sitzungsleitung ausgezählt, was bei einem vollen Hörsaal und klarer Mehrheit auch nicht sinnvoll wäre.
Dieser GO-Antrag soll die Möglichkeit geben, dass jede Person trotzdem eine Zählung fordern kann, wenn sie es für wichtig hält. Dieser Wunsch sollte priorisiert behandelt werden, wie auch die GO-Anträge priorisiert behandelt werden. Das als GO-Antrag zu machen, gibt auch gleich ein klares Handlungsmuster für alle, die eine Zählung wünschen. An anderer Stelle müsste man erst einmal definieren, wann wie so etwas gefordert werden kann.

Die Schreibweise mit den Klammern sorgt vielleicht beim Erstmaligen Lesen für Verwirrung, allerdings wird sie ja offensichtlich (wie man bei dir sehen kann) durch den zweiten Satz aufgeklärt. Zugleich wird die Sitzungsleitung natürlich stets auf die geforderten Quoren hinweisen.

Reply:
Unabhängig von der Diskussion, ob die TO zu Beginn angenommen oder aber (mit Ausnahme ausstehender Änderungsanträge) festgesetzt ist: Hier geht es ausschließlich um die Vertagung. Wenn mehr als die Hälfte der Anwesenden der Meinung ist, dass ein TO vertagt werden sollte, sollte die Minderheit nicht die Macht haben, weiter darüber reden zu lassen. Es bedarf hier auch keinem besonderen Schutz: Offensichtlich sieht sich eine Mehrheit der Anwesenden nicht in der Lage, selbst nach ausführlicher Diskussion eine Entscheidung zu diesem TO zu treffen - sonst würden sie wie von dir beschrieben dafür oder dagegen stimmen. Das allein ist schon Grund genug, den Punkt zu vertagen, um eine unbedachte Entscheidung oder aber die künstliche Verlängerung des TOs zu verhindern